roles/rook/tasks/wait-for-cluster.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Wait 10m until CephCluster is Ready
+  kubernetes.core.k8s_info:
```

Comment:
Missing in osism/zuul-jobs, I will add this there.

Reply:
:+1: